src/uct/ib/base/ib_iface.c

```diff
@@ -1544,6 +1567,7 @@ static UCS_CLASS_CLEANUP_FUNC(uct_ib_iface_t)
         ucs_warn("ibv_destroy_comp_channel(comp_channel) returned %d: %m", ret);
     }
 
+    UCS_STATS_NODE_FREE(self->stats);
```

Comment:
Minor: usually funcs order in dtor is symmetrical (inverse) of the ctor.
In ctor: comp channel, stats, CQ.
Then dtor should be: CQ, stats, comp channel (please move stats `FREE()` above).